### PR TITLE
reformat systemtools

### PR DESCRIPTION
**Pull request by kureta**

### CODE REVIEW

File: supriya/tools/systemtools/SupriyaObject.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 
 class SupriyaObject(AbstractBase):
-    '''Abstract base class from which many custom classes inherit.
-    '''
+    """Abstract base class from which many custom classes inherit.
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/systemtools/SupriyaObject.py]: The regex for adding a newline misses docstrings that do not start with `r`.

File: supriya/tools/systemtools/run_supriya_web.py [hunk 2]
```diff
@@ -25,6 +26,6 @@ def run_supriya_web():
         )
     commands = ' '.join(commands)
 
-    command = r'''python -i {} -c "{}"'''
+    command = r""" python -i {} -c '{}'"""
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/systemtools/run_supriya_web.py]: Just undo this line.

Comment (kureta) [inline supriya/tools/systemtools/run_supriya_web.py]: Also the one in `supriya/tools/systemtools/run_supriya.py` I guess.

**Comment by josiah-wolf-oberholtzer** [comment]
Nice!

**Pull request merged**